docs/shared/packages/expo/expo-plugin.md

```diff
@@ -19,17 +19,24 @@ Make sure to install the `@nx/expo` version that matches the version of `nx` in
 {% /callout %}
 
 {% tabs %}
-{%tab label="npm"%}
+{% tab label="npm" %}
 
 ```shell
-npm i --save-dev @nx/expo
+npm install -D @nx/expo
```

Comment:
Let's use `npm i -D ...` everywhere to make those commands shorter and more readable.

Reply:
Or even better use `npm add -D` since this is the new syntax and it's in sync with all the other package managers.